.circleci/config.yml

```diff
@@ -208,13 +202,7 @@ jobs:
 
 workflows:
   force-build:
```

Comment:
any reason to leave the workflow name as 'force-build' instead of using 'api'?

Reply:
changed to all